File: src/app-upload/sshpass.js
function sshpass(x,...y)
{
 check(is_str,x)
 
 const r=os_execute("sshpass","-p",x,...y)
 const a=split(r)
 
 if(is_full(a))
 { 
  log(...y)
  
  for(const v of a)
  {
   log(" >",v)
  }
 }
 
 return r
}

File: src/prj-core/lib-shell/ssh/ssh-pass.cs
gn ssh_pass x:etc
 let args dup x
 let first shift args

 //the default function is os_execute

 if is_str first
  ret run ssh_pass os_execute x:etc

 //redact

 check is_xn first

 let token shift args

 check is_str token

 let arguments arr "sshpass" "-p" token args:etc

 ret run redact token first arguments:etc
end